src/Controls/src/Core/WebView/WebViewProcessTerminatedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿#nullable disable
+using System;
+
+namespace Microsoft.Maui.Controls
+{
+	public class WebViewProcessTerminatedEventArgs : EventArgs
```

Comment:
Should we add `Sender` like we have for `PlatformPointerEventArgs`?

Reply:
Added.